packages/data-context/src/sources/GitDataSource.ts

```diff
@@ -339,19 +339,22 @@ export class GitDataSource {
 
   async #getInfoPosix (absolutePaths: readonly string[]) {
     debug('getting git info for %o:', absolutePaths)
-    const paths = absolutePaths.map((x) => `"${path.resolve(x)}"`).join(',')
+    const paths = absolutePaths
+    .map((p) => path.resolve(p))
```

Comment:
Any reason we do three `map` here? That's a lot of extra loops.  Can't we do the modification in a single `map`?